iceoryx_hoofs/buffer/include/iox/detail/stack.inl

```diff
@@ -144,10 +144,9 @@ inline optional<T> stack<T, Capacity>::pop() noexcept
         return nullopt;
     }
 
-    // AXIVION Next Construct AutosarC++19_03-A5.2.4 : low level memory management with access to the topmost element on
```

Comment:
The Axivion suppression is not needed here anymore, as the `reinterpret_cast` is being performed inside the `UninitializedArray`.